### PR TITLE
Updated fields API to use LeafUserId as path param

### PR DESCRIPTION
**Pull request by fabyan-alexander**
Update de fields api block:
- POSTs **Create Field 1, Create Field 2,Create Field CFV, Create Field JD, 1**, updated endpoint to include LeafUserId and removed LeafUserId from the request body
- PUT **Update Field**, updated endpoint to include LeafUserId
- DEL **Delete Field**, updated endpoint to include LeafUserId
- GET **Get filter by Id** included
Update de fields/operations api block:
- GET **Get field operations (with filters)**, **Get operations from a field** updated endpoint to include LeafUserId

### CODE REVIEW

File: LEAF_API.postman_collection.json [hunk 1]
```diff
@@ -37,7 +37,10 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n  \"password\": \"{{password}}\",\n  \"rememberMe\": true,\n  \"username\": \"{{username}}\"\n}"
+							"raw": "{\n  \"password\": \"{{password}}\",\n  \"rememberMe\": true,\n  \"username\": \"{{username}}\"\n}",
+							"options": {
+								"raw": {}
+							}
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:43]: Do you know what is this "options"?

File: LEAF_API.postman_collection.json [hunk 2]
```diff
@@ -1577,146 +1658,150 @@
...
+										"type": "text"
+									},
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "https://{{url}}/services/fields/api/users/{{leaf-user-id}}/fields/{{field-1-id}}/operations?sort=id",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:1681]: I think we don't want to let the `sort` parameter in the raw URL. Do we?

File: LEAF_API.postman_collection.json [hunk 3]
```diff
@@ -1577,146 +1658,150 @@
...
+										"{{leaf-user-id}}",
+										"fields",
+										"{{field-1-id}}",
+										"operations"
+									],
+									"query": [
+										{
+											"key": "operationType",
+											"value": "",
+											"description": "Supported values: APPLIED, HARVESTED, PLANTED",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:1700]: Just noticed that we have value "OTHER" too.

File: LEAF_API.postman_collection.json [hunk 4]
```diff
@@ -1902,10 +2002,13 @@
...
...                     41.770789129624966\n                    ],\n                    [\n                        -93.4857443981789,\n                        41.77077578359139\n                    ],\n                    [\n                        -93.48574425678338,\n                        41.77074198964726\n                    ],\n                    [\n                        -93.48574418964974,\n                        41.7707182380936\n                    ],\n                    [\n                        -93.48574413543331,\n                        41.770667665322094\n                    ],\n                    [\n                        -93.48574413543604,\n                        41.77066416532204\n                    ],\n                    [\n                        -93.4857443716175,\n                        41.761630018921785\n                    ],\n                    [\n                        -93.48607005752949,\n                        41.76151767660865\n                    ],\n                    [\n                        -93.4861703248976,\n                        41.761509524885675\n                    ]\n                ]\n            ]\n        ]\n    }\n}\n",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:2005]: externalId shall be removed too :)

File: LEAF_API.postman_collection.json [hunk 5]
```diff
@@ -2256,7 +2341,7 @@
 							}
 						],
 						"url": {
-              "raw": "https://{{url}}/services/fields/api/fields",
+							"raw": "https://{{url}}/services/fields/api/fields?sort",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:2344]: Same thing about the `sort` parameter (?!)

File: LEAF_API.postman_collection.json [hunk 6]
```diff
@@ -2272,65 +2357,100 @@
 									"key": "operationType",
 									"value": "",
 									"description": "Supported values: APPLIED, HARVESTED, PLANTED",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:2359]: Just pointing where `OTHER` shall be included

File: LEAF_API.postman_collection.json [hunk 7]
```diff
@@ -2272,65 +2357,100 @@
...
 									"key": "size",
 									"value": "",
 									"description": "Pagination parameter that specifies the size of each page (integer)",
-                  "disabled": true
+									"disabled": true
 								},
 								{
 									"key": "sort",
-									"value": null,
+									"value": "",
```
Comment (ramnasidharta) [inline LEAF_API.postman_collection.json:2412]: :raised_hands:

**Comment by ramnasidharta** [comment]
[[Docs] Update contents about the new endpoints in dev.agrigate.io](https://trello.com/c/mR7LlX65/292-docs-update-contents-about-the-new-endpoints-in-devagrigateio)

File: LEAF_API.postman_collection.json (hunk 1 above)
Comment (fabyan-alexander) [reply LEAF_API.postman_collection.json:43]: Its strange because there is no options inside a raw body in postman

**Pull request merged**